src/ducks/categories/CategoriesPage.jsx

```diff
@@ -25,6 +29,17 @@ import flag from 'cozy-flags'
 const barTheme = flag('categories-header-primary') ? 'primary' : 'default'
 
 class CategoriesPage extends Component {
+  constructor(props) {
+    super(props)
+    const { filteringDoc, resetFilterByDoc } = this.props
+    if (
+      filteringDoc &&
+      includes(['Reimbursements', 'health_reimbursements'], filteringDoc._id)
+    ) {
+      resetFilterByDoc()
+    }
+  }
+
```

Comment:
@ptbrowne had a good example of a component that can be constructed but never mounted (which I can't remember, sorry). With this in mind, I think this logic should be put in a `componentDidMount` method.

Reply:
```jsx
const a = <A />
if (b) { reutrn null}
return a
```

le composant A est instancié mais pas monté